egs2/chime7_task1/asr1/local/generate_chime6_data.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# Copyright 2019, Johns Hopkins University (Author: Shinji Watanabe)
+# Apache 2.0
+#
+# This script generates synchronized audio data across arrays by considering
+# the frame dropping, clock drift etc. done by Prof. Jon Barker at University of
+# Sheffield. This script first downloads the synchronization tool and generate
+# the synchronized audios and corresponding JSON transcription files
+# Note that
+# 1) the JSON format is slightly changed from the original CHiME-5 one (simplified
+# thanks to the synchronization)
+# 2) it requires sox v.14.4.2 and Python 3.6.7
+# Unfortunately, the generated files would be different depending on the sox
+# and Python versions and to generate the exactly same audio files, this script uses
+# the fixed versions of sox and Python installed in the miniconda instead of system ones
+# copied from https://github.com/kaldi-asr/kaldi/blob/master/egs/chime6/s5_track1/local/generate_chime6_data.sh
+
+. ./cmd.sh
+. ./path.sh
+
+# Config:
+cmd=run.pl
+
+. utils/parse_options.sh || exit 1;
+
+if [ $# != 2 ]; then
+  echo "Wrong #arguments ($#, expected 2)"
+  echo "Usage: local/generate_chime6_data.sh [options] <chime5-in-dir> <chime6-out-dir>"
+  echo "main options (for others, see top of script file)"
+  echo "  --cmd <cmd> # Command to run in parallel with"
+  exit 1;
+fi
+
+sdir=$1
+odir=$2
+expdir=${PWD}/exp/chime6_data
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# get chime6-synchronisation tools
+SYNC_PATH=${PWD}/chime6-synchronisation
+if [ ! -d ${SYNC_PATH} ]; then
+  git clone https://github.com/chimechallenge/chime6-synchronisation.git
+fi
+
+mkdir -p ${odir}
+mkdir -p ${expdir}/log
+
+# split the session to avoid too much disk access
+sessions1="S01 S02 S03 S04 S05 S06 S07"
+sessions2="S08 S09 S12 S13 S16 S17 S18"
+sessions3="S19 S20 S21 S22 S23 S24"
+
+CONDA_PATH=${MAIN_ROOT}/tools/venv/bin/sox
```

Comment:
In my local environment, this should be `CONDA_PATH=${MAIN_ROOT}/tools/venv/envs/espnet/bin`. Is the last `sox` required in your environment?

Reply:
I should ve changed this part https://github.com/espnet/espnet/pull/4934